playbooks/git.yml

```diff
@@ -0,0 +1,20 @@
+---
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+
+- name: build the git vcs
```

Comment:
```suggestion
- name: build our in-house GitLab instances
```

Reply:
I would still like to see this name changed. When I see `build the git vcs` I don't have an immediate understanding of what is happening.

Reply:
I would also like to see the name of the playbook file changed from `git.yml` to `GitLab.yml`.